Several targets on the Makefile not working properly

**Describe the bug**
The following targets on the Makefile, will cause an error when trying to run them individually:
- start-db
- stop-db
- start-dmsg
- stop-dmsg
- attach-dmsg
- start-pty
- stop-pty


**Environment information:**

- OS: Ubuntu 19.10
- Platform: Linux 5.3.0-64-generic x86_64

**Steps to Reproduce**
Testing `start-db` target. The same can be achieved with the others targets:
1. Start a redis database on the default port
2. Run `make start-db`
3. See error

**Actual behavior**
Here is the result after running the target:
```
source ./integration/env.sh && init_redis
/bin/sh: 1: source: not found
make: *** [Makefile:82: start-db] Error 127
```
At first I thought the file `./integration/env.sh` was not placed on the working directory, but that's not the case.

**Additional context**
Add any other context about the problem here.

**Possible implementation**
After reading these:
1. [Choosing the Shell(GNU)](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html)
2. [Source command not found](https://stackoverflow.com/questions/13702425/source-command-not-found-in-sh-shell)

I noticed that given the current **Makefile**, the value of **`SHELL`** is `/bin/sh`, which is the default value for **`SHELL`** and source is not supported by this shell.
A possible solution would be:

```
ifneq ($(OS), Windows_NT)
        UNAME_S := $(shell uname -s)
        ifeq ($(UNAME_S), Linux)
                # supporting source command otherwise /bin/sh
                # will be taken as default, which doesn't support source
                SHELL := /bin/bash
        endif
endif
```
At the start of the Makefile.

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+SHELL := /bin/bash
+
 .DEFAULT_GOAL := help
 .PHONY : check lint install-linters dep test build
 

--- a/integration/env.sh
+++ b/integration/env.sh
@@ -173,7 +173,7 @@ function init_dmsg() {
   func_print "Running ${DMSG_SRV2}..."
   tmux send-keys -t ${DMSG_SRV2} './bin/dmsg-server ./integration/configs/dmsgserver2.json' C-m
   catch_ec $?
-
+  sleep 1
   func_print "${DMSG} session started successfully."
   tmux select-window -t bash
 }

